Resources/Locale/en-US/markings/tattoos.ftl

```diff
@@ -1,8 +1,14 @@
 marking-TattooHiveChest-tattoo_hive_chest = Back Tattoo (Hive)
 marking-TattooHiveChest = Back Tattoo (Hive)
 
-marking-TattooNightlingChest-tattoo_nightling = Chest Tattoo (nightling)
-marking-TattooNightlingChest = Chest Tattoo (Nightling)
```

Comment:
Leave this alone, move the female variants to deltav namespace

Reply:
There's a modification to the male one, it had a pixel peeking past the body. Should I move both variants?